### ISSUE TITLE
[SUGGESTION]New Ring with slots for all other rings

### ISSUE THREAD

**Issue by CaioCPS** [Ellpeck/ActuallyAdditions#396]
My idea is, make a ring with slots to contain all rings that have effects, thus leaving the inventory cleaner

**Comment by bookerthegeek** [comment]
Better yet make them baubles

**Comment by Ellpeck** [comment]
No baubles.

**Comment by PapiDimmi** [comment]
You should really add Baubles support, Ellpeck.

**Comment by Ellpeck** [comment]
I don't care what I should, I said multiple times before that I am not willing to add Baubles support.

**Comment by PapiDimmi** [comment]
All right, then. I’m just saying that it would be neat, but that’s your choice.

Would you mind telling me why you don’t want to add Baubles support?

Anyway, maybe you should add a special bag that holds all your rings so they don’t fill up your inventory.

**Comment by Ellpeck** [comment]
I don't like Baubles, that's as easy as the explanation gets.
Also, this issue is about a ring that holds all other rings, so I don't understand why you're suggesting a bag.

**Comment by PapiDimmi** [comment]
Yeah, I’m agreeing with @CaioCPS; a ring or bag that holds other rings would be useful.

**Comment by chimericdream** [comment]
tl;dr
Love this idea. 👍  from me.

---

I love the idea of a "master ring" that combines the effects of the others. In fact, I came to this issue tracker planning to suggest this exact feature (if it hadn't been already, which obviously... here I am).

Some ideas for how I could see it working: it could be a container with slots for rings, or possibly a basic shapeless recipe where you craft rings together to get a master ring with the combined effects. It seems to me the shapeless recipe would be easier from a code standpoint, since you'd just combine the NBT data from each of the rings used. But it all depends what you're going for from a flavor perspective.

If you want to gate the master ring behind something that requires a little more effort and/or make it more than just an item to reduce the number of inventory slots used (which is already enough, but again, depends what you're shooting for in the "lore"), that could be interesting as well. Maybe increase the bonuses, give some kind of set bonus, something. If you went that route, I could definitely see the master ring requiring more advanced crafting materials, like blocks of empowered *(insert crystal type here)* or something.

I've got a few other ideas for how this could work, but this comment is already long enough. If you're interested, I can post them. If not, no worries.

**Comment by Ellpeck** [comment]
_I've decided to stop taking suggestions on this issue tracker. As I don't work on Actually Additions myself anymore and Shadows only maintains the mod, I don't see why they would stick around here cluttering it even though they'll probably never see the light of day._
